client/my-sites/email/email-management/home/email-plan/index.jsx

```diff
@@ -59,7 +62,8 @@ const UpgradeNavItem = ( { currentRoute, domain, selectedSiteSlug } ) => {
 			path={ emailManagementPurchaseNewEmailAccount( selectedSiteSlug, domain.name, currentRoute ) }
 			onClick={ () => recordTracksEvent( 'calypso_upsell_email', { context: 'email-forwarding' } ) }
 		>
-			{ translate( 'Upgrade to a hosted email' ) }
+			{ translate( 'Upgrade to Professional Email' ) }
+			<Badge type="info-green">{ translate( 'Try 3 months free' ) }</Badge>
```

Comment:
@nightnei We need to check if the domain is still eligible for the free trial. Imagine the case where a user
* subscribed to Professional Email with this domain
* cancelled their subscription
* setup email forwarding with this domain

In this case, they will no longer be eligible for the introductory offer.

Reply:
Thank you @wongasy 
Fixed it and updated test instructions